crates/core/app/src/penumbra_host_chain.rs

```diff
@@ -0,0 +1,50 @@
+use std::str::FromStr;
+
+use anyhow::Context;
+use async_trait::async_trait;
+use ibc_types::core::connection::ChainId;
+use penumbra_chain::{component::StateReadExt, state_key};
+use penumbra_ibc::component::HostInterface;
+use penumbra_proto::StateReadProto;
+use tendermint::Time;
+
+// ibc-async HostChainInterface for the Penumbra chain
```

Comment:
i think this comment refers to a symbol that was renamed? it would be nice, but not blocking, to document this `PenumbraHost` type lightly, since it is publicly exported by the `penumbra-app` crate.